.github/workflows/test.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       max-parallel: 20
       matrix:
         os: [ubuntu-latest, macos-14, windows-latest]
-        python-version: ["3.9", "3.x"]
```

Comment:
Looks like setup action [sometimes install python 3.12](https://github.com/materialsvirtuallab/monty/actions/runs/11432932326/job/31804141443), and sometimes 3.13, [if 3.13 is installed](https://github.com/materialsvirtuallab/monty/actions/runs/11438818081/job/31821218765?pr=713), would trigger the following error on MacOS:
```
ERROR: Ignored the following versions that require a different python version: 1.21.2 Requires-Python >=3.7,<3.11; 1.21.3 Requires-Python >=3.7,<3.11; 1.21.4 Requires-Python >=3.7,<3.11; 1.21.5 Requires-Python >=3.7,<3.11; 1.21.6 Requires-Python >=3.7,<3.11; 1.26.0 Requires-Python <3.13,>=3.9; 1.26.1 Requires-Python <3.13,>=3.9
ERROR: Could not find a version that satisfies the requirement torch; extra == "json" (from monty[dev,json,multiprocessing,serialization]) (from versions: none)
ERROR: No matching distribution found for torch; extra == "json"
```